pkg/networkmanager/interfaces.js

```diff
@@ -2144,6 +2145,11 @@ PageNetworkInterface.prototype = {
             cockpit.location.go('/');
         });
 
+        $("#networking-firewall-link, #networking-firewall-summary").on("click", function() {
+            cockpit.jump("/network/firewall");
```

Comment:
Indeed it is not necessary any more to specify the host in master, but in the interest of backportability, and for working with older cockpit-system in RHEL, please add it here.